query-node/schemas/content.graphql

```diff
@@ -72,6 +72,9 @@ type Channel @entity {
 
   "List of channel collaborators (members)"
   collaborators: [Membership!]
+
+  #"List of biggest video NFT owners."
+  #nftCollectors: [Membership!]!
```

Comment:
Should this be commented out?

Reply:
These commented lines have been removed, and proper relationship has been created here.